packages/utils/src/signatureUtils.ts

```diff
@@ -25,6 +27,34 @@ export interface Signer {
   sign: (message: string) => Promise<string>
 }
 
+export async function addressToPublicKey(
+  signer: string,
+  signFn: (message: string, signer: string) => Promise<string>
+) {
+  const msg = new Buffer('dummy_msg_data')
+  const data = '0x' + msg.toString('hex')
+  // Note: Eth.sign typing displays incorrect parameter order
+  const sig = await signFn(data, signer)
+
+  const rawsig = ethjsutil.fromRpcSig(sig)
+  const prefixedMsg = hashMessageWithPrefix(data)
+  const pubKey = ethjsutil.ecrecover(
+    Buffer.from(prefixedMsg.slice(2), 'hex'),
+    rawsig.v,
+    rawsig.r,
+    rawsig.s
+  )
+
+  const computedAddr = ethjsutil.pubToAddress(pubKey).toString('hex')
+  assert(eqAddress(computedAddr, signer), 'computed address !== signer')
+
+  return '0x' + pubKey.toString('hex')
+}
+
+export function eqAddress(a: Address, b: Address) {
```

Comment:
There's an `areAddressesEqual` below in this file AND @yorhodes adds another one in his PR here: #1570 
I'll leave it to you two to coordinate but pretty sure we don't need another util method for this ;)

Reply:
Must have been a merge conflict? This was supposed to move to address.ts